charts/redis-ha/values.yaml

```diff
@@ -544,7 +544,8 @@ exporter:
     periodSeconds: 15
     successThreshold: 2
 
-podDisruptionBudget: {}
+podDisruptionBudget:
+  {}
```

Comment:
Can we undo the newlines on `[]` and `{}` based values?

Reply:
Oh
That was my linter
Of course we can undo these

Reply:
That should be okay now